### PR TITLE
Use simplified simpleicons for social-media

### PR DESCRIPTION
**Pull request by kfessel**
applied https://jakearchibald.github.io/svgomg/ 
matrix icon is node reduced

### CODE REVIEW

**Comment by leandrolanzieri** [comment]
Do you think we could have the svg files separately? Also, for some reason in the bigger version of the social media bar the matrix logo is too close to the border, which does not happen in the lower one.

**Comment by kfessel** [comment]
I like the inline svg (may load faster than 6 requests for 300 to 1000Byte).
the circle (it is a sqaure with round corners) touches in one and not in the other version because it has a margin in px.

File: _sass/pages/index.scss [hunk 1]
```diff
@@ -201,26 +201,33 @@ section {
 # social media
 --------------------------------------------------------------*/
 
-#social-media .social-links a {
+.social-links a {
```
Comment (leandrolanzieri) [inline _sass/pages/index.scss:204]:
```suggestion
.social-links  {
  svg{
    height:1em;
    width:1em;
    margin: auto;
    display: block;
  }

  a {
    display: inline-block;
    background: transparent;
    color: $light;
    line-height: 1;
    padding: 0.4em;
    margin-right: 0.1em;
    border-radius: 50%;
    text-align: center;
    width: 1.8em;
    height: 1.8em;
    transition: 0.3s;

    &:hover {
      background: $primary;
      color: $light;
      text-decoration: none;
    }
  }
```

**Comment by leandrolanzieri** [comment]
Also this needs rebase now

**Comment by kfessel** [comment]
rebased and nested

**Review by leandrolanzieri — APPROVED** [review]
Thanks for this!

**Comment by miri64** [comment]
Mhh with this PR the discourse link shows now a different logo (cmp https://riot-os-riot-os-org-preview-98.surge.sh/ vs current riot-os.org) and the mouse-over says "Disqus", which is a [completely different service](https://disqus.com/profile/login/) from Discord.

**Comment by leandrolanzieri** [comment]
Good catch, I did not notice that. We should use https://simpleicons.org/?q=discourse

**Comment by kfessel** [comment]
Isnt out forum "Disqus"

**Pull request merged**